src/parser/deserializer.ts

```diff
@@ -120,6 +123,47 @@ function deserializeObject(
   const promoteLongs = options['promoteLongs'] == null ? true : options['promoteLongs'];
   const promoteValues = options['promoteValues'] == null ? true : options['promoteValues'];
 
+  // Ensures default validation option if none given
+  const validation = options['validation'] == null ? { utf8: true } : options['validation'];
```

Comment:
```suggestion
  const validation = options.validation == null ? { utf8: true } : options.validation;
```

Reply:
Should I change this for the other options as well?

Reply:
Sure! that would be a nice improvement